src/main/resources/log4j2-spring.xml

```diff
@@ -10,6 +10,7 @@
         </Console>
     </Appenders>
     <Loggers>
+        <Logger name="org.zalando.logbook.Logbook" level="TRACE"/>
```

Comment:
i wouldn't enable it forever, but for monitoring purposes we should have a better insight in, what goes through the ether ;)
And to quieten you regarding the CPU problem:

> 4. Using unique sourcetype filed in job file + Not removing watermark file — Reduced the CPU utilisation (Tested with 1, 100, 500, 1500 loglines/sec)

> Now, we know the root cause of the problem and know how to solve this (by using a unique “sourcetype" value in the job file).

There was an email yesterday where Rajat explained how to solve that CPU utilization problem.